feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/TopicScreen.kt

```diff
@@ -114,8 +86,12 @@ internal fun TopicScreen(
                 )
             }
 
-            TopicUiState.Error -> TODO()
-            is TopicUiState.Success -> {
+            Error -> {
+                item {
+                    Text(text = "error")
```

Comment:
I think this is still WIP for the error case?

Reply:
It is! I didn't change here anything, it's still a WIP on the main branch as well. I would gladly address this, but prefer to do it as a separate PR. Is that ok?